app/scripts/controllers/recent-blocks.js

```diff
@@ -1,11 +1,13 @@
 const ObservableStore = require('obs-store')
 const extend = require('xtend')
+const BN = require('ethereumjs-util').BN
 
 class RecentBlocksController {
 
   constructor (opts = {}) {
-    const { blockTracker } = opts
+    const { blockTracker, ethQuery } = opts
```

Comment:
prefer `provider` over `ethQuery` (ethQuery is a stateless helper around the provider interface)

Reply:
Even tho ethQuery is already being initialized in mm-controller and passed to a controller?

Would you say that controller should also prefer provider?